app/views/errors/not_found.html.erb

```diff
@@ -0,0 +1,3 @@
+<!-- created view templates in app/views/errors/ for the not found (404) page -->
+<h1>The page you were looking for could not be found.</h1>
+<p>Please check the URL to make sure it is correct. If it is correct, the administrators will be notified that there is an issue with this page.</p>
```

Comment:
can you remove this section, we won't be notified when there is a 404 error `If it is correct, the administrators will be notified that there is an issue with this page.`